drivers/clock_control/clock_stm32l0_l1.c

```diff
@@ -26,29 +25,59 @@
 #define pll_div(v) z_pll_div(v)
 
 /**
- * @brief Set up pll configuration
+ * @brief Return PLL source
  */
-int config_pll_sysclock(void)
+__unused
+static uint32_t get_pll_source(void)
 {
-	uint32_t pll_source, pll_mul, pll_div;
-
-	pll_mul = pll_mul(STM32_PLL_MULTIPLIER);
-	pll_div = pll_div(STM32_PLL_DIVISOR);
-
 	/* Configure PLL source */
 	if (IS_ENABLED(STM32_PLL_SRC_HSI)) {
-		pll_source = LL_RCC_PLLSOURCE_HSI;
+		return LL_RCC_PLLSOURCE_HSI;
 	} else if (IS_ENABLED(STM32_PLL_SRC_HSE)) {
-		pll_source = LL_RCC_PLLSOURCE_HSE;
-	} else {
+		return LL_RCC_PLLSOURCE_HSE;
+	}
+
+	return -ENOTSUP;
```

Comment:
it is not correct to return -ENOTSUP as return type is uint32_t.
Maybe #ifdef and #error combination is better?

Reply:
This is indeed one part that requires more thinking.
Switched to asserts